src/styles/blocks/card.scss

```
@@ -0,0 +1,77 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: $light;
+  border: 1px solid $border-light;
+  border-radius: 5px;
+ 
+  &__body {
+    padding: 32px 16px 16px;
+  }
+
+  &__img {
+    margin-bottom: 36px;
+    text-align: center;
+
+    img {
+      height: 134px;
+    }
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__text {
+    margin-bottom: 16px;
+    color: $secondary;
+  }
+
+  &__inner {
+    @extend %flex-space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    @extend %flex-space-between;
+    margin-bottom: 16px;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary;
+
+    span {
+      font-size: 16px;
+      font-weight: 700;
+      color: $dark;
+    }
+  }
```

Add class to this element in html and use it in scss.

Example of code
```suggestion
  &__price {
    @extend %flex-space-between;
    margin-bottom: 16px;
    font-size: 12px;
    line-height: 18px;
    color: $secondary;
  }
  
  &price-tag {
      font-size: 16px;
      font-weight: 700;
      color: $dark;
    }
```